package.json

```diff
@@ -47,7 +47,7 @@
     "npm-run-all": "^4.1.5",
     "pngjs": "^7.0.0",
     "prettier": "^2.8.8",
-    "three": "^0.153.0",
+    "three": "^0.151.0",
```

Comment:
Unexpected change in r151 → r152.

I'll investigate this in the other PR.

Terrarian viewer in r151:

![Terrarian Viewer in r151](https://github.com/kui/7dtd-map/assets/29692/431b4d53-3795-40c1-83fa-9a1eeba297a3)

Terrarian viewer in r152:

![Terrarian Viewer in r152](https://github.com/kui/7dtd-map/assets/29692/4a9fd85d-b4d5-446b-9e87-3c8234583337)